modules/runtime-config/utils/getRuntimeConfigValue.ts

```diff
@@ -0,0 +1,11 @@
+import IS_SERVER from '#ioc/config/IS_SERVER'
+import runtimeConfig from '~/.sfx/runtime/config'
+
+export default <Key extends keyof typeof runtimeConfig>(key: Key): typeof runtimeConfig[Key] => {
+  if (IS_SERVER) {
```

Comment:
Maybe consider adding some kind of error handling here (can be optional / only for non-production env etc.) to know from the console which variables are used but missing.
Also, a fallback/default value passed as the second param could be useful.

Reply:
Error handling is tricky here. Currently this function is type-safe is TypeScript will complain if you try to access non-existent runtime config (autcomplete for all possible runtime configs also works). But what about runtime? I guess adding dev-only warning about accessing non-existent runtime configs couldn't hurt, but thats probably it. We could also throw an error if runtime config does not exist AND default value was not provided, but I think this functionality is better suited for some other helper function or override. 

About the default value. Currently you can do it with with `??` operator like this: `getRuntimeConfigValue('FOO') ?? 'BAR'`. We could add support for default value via second parameter, but question is if it has any upsides (except for possibility of some more elaborate error handling).